flink-connectors/flink-connector-cassandra/src/main/java/org/apache/flink/streaming/connectors/cassandra/CassandraSink.java

```diff
@@ -95,6 +95,28 @@ private CassandraSink(SingleOutputStreamOperator<IN> sink) {
 	}
 
 	/**
+	 * Sets an additional, user provided hash for this operator.
+	 *
+	 * <p/>
+	 * <p>The user provided hash is an alternative to the generated hashes, that is considered when identifying an
+	 * operator through the default hash mechanics fails (e.g. because of changes between Flink versions.
+	 *
+	 * <p><strong>Important</strong>: this hash needs to be unique per transformation and job. Otherwise, job
+	 * submission will fail.
+	 *
+	 * @param hash the user provided hash for this operator.
+	 * @return The operator with the user provided hash.
+	 */
+	public CassandraSink<IN> provideAdditionalNodeHash(String hash) {
```

Comment:
I think the `provide` prefix gives the impression that it is possible to provide multiple additional node hashes. I would replcae this with `set`. What do you think?